scripts/compute_DICE.sh

```diff
@@ -14,8 +14,14 @@ then
 lower_thr=1
 upper_thr=100000 # Just pick a large number
 else
+if [ "$#" -eq 3 ]
+then
 lower_thr=$3
 upper_thr=$3
+else
```

Comment:
add to the heading that the script can also take in two number inputs after the two volumes if you want to compare across ROIs